compute/instances/concepts.mdx

```diff
@@ -238,15 +237,15 @@ Stand-by mode is designed to temporarily stop an instance. When you put a comput
 
 ## Stateful Security Groups
 
-Stateful Security Groups disregard the default policy and inbound/outbound rules if a connection is initiated from your instance. Traffic will always be permitted on connections that you initiated. This is useful for example if you want to initiate connections on many different and various ports for messaging, video streaming or other purposes. In this case, the traffic from/to that connection will not be blocked, even if you have defined a rule that would otherwise do so.
-
+Stateful Security Groups disregard the default policy and inbound/outbound rules if a connection is initiated from your instance. Traffic will always be permitted on connections that you initiated. This is useful for example if you want to initiate connections on many different and various ports for messaging, video streaming or other purposes. In this case, the traffic from/to that connection will not be blocked, even if you have defined a rule that would otherwise do so. Find out more with our dedicated documention: [how to use Security Groups](/compute/instances/how-to/use-security-groups/)
```

Comment:
```suggestion
Stateful Security Groups disregard the default policy and inbound/outbound rules if a connection is initiated from your Instance. Traffic will always be permitted on connections that you initiated. This is useful for example if you want to initiate connections on many different and various ports for messaging, video streaming, or other purposes. In this case, the traffic from/to that connection will not be blocked, even if you have defined a rule that would otherwise do so. Find out more with our dedicated documentation: [how to use Security Groups](/compute/instances/how-to/use-security-groups/)
```